src/main/scala/com/typesafe/sbt/packager/archetypes/JavaServerApplication.scala

```diff
@@ -284,6 +284,8 @@ object JavaServerAppPackaging extends AutoPlugin {
     Some(script)
   }
 
+  protected def makePostInstallScript(): Option[String] = ???
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Usage of ??? operator](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/1241567461/issues/source?bid=2040925&fileBranchId=2935951#l287)